src/core/annotation.js

```diff
@@ -378,7 +370,9 @@ var Annotation = (function AnnotationClosure() {
       }
       opList.addOp(OPS.endAnnotations, []);
       annotationsReadyCapability.resolve();
-    }, reject);
+    }, function(reason) {
+      annotationsReadyCapability.reject(reason);
+    });
```

Comment:
Nice suggestion! I have done this in the new commit, also with more descriptive names (as `datas` was a bit vague in my opinion).